libs/ha/ha_common_libs_k8s.py

```diff
@@ -892,11 +893,12 @@ def event_s3_operation(self, event, log_prefix=None, s3userinfo=None, skipread=F
             workloads.extend(HA_CFG["s3_bench_large_workloads"])
         # Flag to store next workload status after/while event gets clear from test function
         event_clear_flg = False
-        resp = s3bench.setup_s3bench()
-        if not resp:
-            status = (resp, "Couldn't setup s3bench on client machine.")
-            output.put(status)
-            sys.exit(1)
+        if setup_s3bench:
```

Comment:
so we will check from test if we want to setup? 
I guess we had discussion and to avoid working with non-proper version, we decided not to check if s3becnh installed, is that right?

Reply:
We decided to go for installation in first process in test, and for 2nd process we can keep installation False. So that we will get latest version of s3bench while running test.
This flag will do that only, i am not checking if s3bench is installed or not. If this flag the True (Default is True) is will go and install whichever available version.